### ISSUE TITLE
Error: Cannot find module 'opn'

### ISSUE THREAD

**Issue by amannn** [micahblu/gulp-connect-php#1]
Could it be that `opn` should be moved from devDependancies to dependancies?

After installing gulp-connect-php I get this error when running a gulp task that uses gulp-connect-php:

```
[17:53:27] Error: Cannot find module 'opn'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (**project-folder**/node_modules/gulp-connect-php/index.js:4:12)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
```

**Comment by michaelgrossklos** [comment]
gulp-connect-php has two dependencies: 
1. opn 
2. bin-version-check

install these via `nmp install --save-dev opn bin-version-check` and you're good to go ;-)

**Comment by LantosIstvan** [comment]
Why not just simply removing this line from '.npmignore' file:

```
node_modules/
```

Is there any reason it's need to be in there?

**Comment by amannn** [comment]
I guess ignoring the folder `node_modules` is fine in order to prevent bloating the repo with unnecessary files.

I just had a quick look at how other gulp plugins are doing it. For example [gulp-sass](https://github.com/dlmanning/gulp-sass) has all the dependancies that the plugin needs to be used in a gulp task in its `dependancies` and therefore it will work perfectly if you do `npm install gulp-sass` and use it in a gulp task.

Its `devDependancies` are only used if you want to run tests against the plugin.

Isn't this the way it should be?

**Comment by LantosIstvan** [comment]
Well, every single Node.js repo bloating with lots of other repos... :) Except TotalJS.

Dependencies are neccesary to use this plugin, that's why I don't understand why uncommented them in .npmignore, by default. Without 'opn' the server simply not starting. Later on I have to manually install dependencies,  going into the plugin's folder.

**Comment by wrouesnel** [comment]
+1 that this needs to be fixed. If I want to deduplicate my dependencies I'll do it manually on my end somehow. What I want from npm is to be able to hand this off to another developer and say "run npm install" to get started.

EDIT: at the very least this needs to be made more obviously explicit. The module name "opn" looks exactly like a typo in context when you examine the source.

**Comment by micahblu** [comment]
Sorry all, I received no emails on these issues? Just now seeing them. I'm going to look into this tonight and hopefully have a patch soon

**Comment by micahblu** [comment]
This should be fixed as of 0.0.3, thanks to Endel
